### DISCUSSION TITLE
Import JS Library in UDF

### DISCUSSION THREAD

**Discussion by exwhyz** [crate/crate#11059] [General] [answered]
Is it possible to import a JavaScript file/module/library as part of a UDF? If not are there any plans to support this in the future?

**Comment by proddata** [comment, marked as answer]
Hi @exwhyz 

Unfortunately this is not possible right now and I am not aware of any concrete plans to support this.
Could you elaborate what libraries would be of an interest to you? Maybe we could add some libraries of common interest.

regards
Georg

**Reply by exwhyz** [reply]
Hi @proddata - Thanks for the reply. These are 2 libraries in the order of preference primarily focused on statistical functions. 
https://github.com/jstat/jstat
https://github.com/stdlib-js/stdlib

Thanks!
